subjects/sortedlistmerge/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"piscine"
+)
+
+type NodeL struct {
+	Data interface{}
+	Next *NodeL
+}
+
+type List struct {
```

Comment:
This type is not used in this code snippet.

Reply:
removed